vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_util.h

```diff
@@ -17,8 +17,11 @@ namespace ml {
 namespace pipeline {
 
 struct PipelineInfo;
+struct PipelineEmbeddingInfo;
```

Comment:
Please sort alphabetically, please check other forward decl for this PR

Reply:
Sorted